doc/genmodel.py

```diff
@@ -388,12 +397,18 @@ def process_model(py_file, force=False):
     }
 
     # Generate the RST file and the figure.  Order doesn't matter.
-    print("generating", rst_file)
+    print("generating rst", rst_file)
+    print("1: docs")
     gen_docs(model_info, rst_file)
+    print("2: figure", end='')
     if force:
+        print()
```

Comment:
Do we need empty print?

Reply:
It adds the new-line omitted in the previous print statement.